comment/migrations/0009_auto_20200811_1945.py

```diff
@@ -7,8 +7,9 @@
 def set_default_email(apps, schema_editor):
     comment_model = apps.get_model('comment', 'Comment')
     for comment in comment_model.objects.all():
-        comment.email = comment.user.email
-        comment.save(update_fields=['email'])
+        if hasattr(comment.user, 'email'):
+            comment.email = comment.user.email
+            comment.save(update_fields=['email'])
```

Comment:
This change will require an additional test case in the [`test_migrations` module](https://github.com/radi85/comment/blob/master/comment/tests/test_migrations.py). The migration test may turn out to be difficult(because of the way they are written and how they run) to deal with at this moment. Feel free to play with them if you want to.

OR 

You can choose to remove this change, I will try to take care of this in the future.

Reply:
Why do we need to check if the user has an email field? I believe the email field will be always an attribute of the user object since it is already an attribute of django abstract user model. So even if someone subclass django user model and didn't explicitly use the email field, it will still be an attribute of the user object with a blank value.
So I believe this check statement is not needed and it can be removed from all occurrences in the code.

Reply:
> I believe the email field will be always an attribute of the user object since it is already an attribute of django abstract user model

No, developers can delete this field using `model._meta` method. Also, one more thing `email` and `username` field shouldn't be hard coded as developers can use their own name for this. We should be using `user_model.USERNAME_FIELD` and `user_model.EMAIL_FIELD` everywhere.

You can see the relevant documentation regarding custom user model here:
https://docs.djangoproject.com/en/dev/topics/auth/customizing/#django.contrib.auth.models.CustomUser

Reply:
That is why I wanted the author to ignore these changes, and I planned to incorporate these in a new one.

Reply:
Nice observation, we need to open a new ticket to adjust all occurrences of `user.email` and `user.username`.